mage_ai/ai/llm_pipeline_wizard.py

```diff
@@ -294,6 +301,7 @@ async def async_generate_block_with_description(
             block_description: str,
             upstream_blocks: List[str] = None) -> dict:
         messages = [{"role": "user", "content": block_description}]
+        # TODO(Replace with a generic LLM call so all LLM clients can support this funciton.)
```

Comment:
So does it still require openai now?

Reply:
Yes, because the `openai.ChatCompletion.acreate` usage here, so the hugging face client still requires openai key available. I have the openai key read for hugging face client at line 199. I will remove both TODO once this open ai function call being migrated to client.